src/django_components/templatetags/component_tags.py

```diff
@@ -298,29 +797,22 @@ class ParsedTag(NamedTuple):
 
 
 def _parse_tag(
```

Comment:
I might move `_parse_tag` to a sepate file in `util/` eventually, but not in this MR for simplicity's sake.